bin/cylc

```diff
@@ -27,8 +27,8 @@ get_version() {
         CYLC_VERSION="$(cat "${CYLC_DIR}/VERSION")"
     # Otherwise, use git (if in a git repo and not a bare clone)
     # to determine version
-    elif git -C "${CYLC_DIR}" rev-parse && \
-            [[ -n "$(git -C "${CYLC_DIR}" rev-parse --show-toplevel)" ]]; then
+    elif git --git-dir "${CYLC_DIR}/.git" rev-parse && \
+            [[ -n "$(git --git-dir "${CYLC_DIR}/.git" rev-parse --show-toplevel)" ]]; then
```

Comment:
If I was going to really pedantically review my own code - this line is now longer than 80 characters and breaks the Google shell script style guidelines...